main/integration-tests/__test__/api-tests/projects/create-project.test.js

```diff
@@ -0,0 +1,67 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('Create project scenarios', () => {
+  let setup;
+  let adminSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('Creating a project', () => {
+    it('should fail if admin is inactive', async () => {
+      const testProjectId = setup.gen.string({ prefix: `inactive-admin-create-proj-test` });
+      const admin2Session = await setup.createAdminSession();
+      await adminSession.resources.users.deactivateUser(admin2Session.user);
+
+      await expect(admin2Session.resources.projects.create(testProjectId)).rejects.toMatchObject({
+        code: errorCode.http.code.unauthorized,
+      });
+    });
+
+    it('should fail if non-admin user is trying to create project', async () => {
+      const researcherSession = await setup.createResearcherSession();
+
+      await expect(researcherSession.resources.projects.create(setup.gen.defaultProjectId())).rejects.toMatchObject({
+        code: errorCode.http.code.forbidden,
+      });
+    });
+
+    it('should fail if projectId is duplicate to the one already in the system', async () => {
+      const admin2Session = await setup.createAdminSession();
+
+      await expect(admin2Session.resources.projects.create(setup.gen.defaultProjectId())).rejects.toMatchObject({
+        code: errorCode.http.code.badRequest,
+      });
+    });
+
+    it('should fail for anonymous user', async () => {
```

Comment:
Not directly related to this PR, just a thought - maybe we can create the anonymous user test on a higher level instead of individual test classes? Since each non-public API endpoint need the anonymous user test, and they should reply the same error code.

Reply:
Yeah, we'll then need to think of how to send `<resource>.<operation>()` dynamically to this test case. Something for the future..